debian/changelog

```diff
@@ -1,3 +1,65 @@
+ubuntu-advantage-tools (27.11~22.10.1) kinetic; urgency=medium
+
+  * d/control:
+    - Update VCS references
+  * d/links:
+    - add usr/bin/pro as an alias to ubuntu-advantage
+  * d/postinst:
+    - include root_mode parameter when creating UAConfig instances
+    - change calls to add_notice to notice_file.add
+    - create public machine-token file if it does not exist
+  * New upstream release 27.11 (LP: #1989279)
+    - api:
+      + new `pro api` command to access the public client API
+      + 'version' endpoint returning version information
+      + 'should auto attach' endpoint informing if a system should run
+        auto-attach on startup
+      + 'full auto attach' endpoint performing auto-attach
+      + 'magic attach' endpoints for the Magic Attach flow
+    - auto-attach:
+      + better errors for invalid pro images (GH: #2180, #1833)
+      + don't detach on already auto-attached instances
+      + no-op when ubuntu-advantage information is present on cloud-init userdata
```

Comment:
Lintian is stating that this line is too long

Reply:
good catch